pycbc/results/pygrb_postprocessing_utils.py

```diff
@@ -570,6 +584,124 @@ def extract_ifos_and_vetoes(trig_file, veto_files, veto_cat):
+    for param in rec_params:
+        found_data['rec_%s' % param] = inj_data['/network/%s' % param][...]
+
+    found_data['end_time'] = \
+        inj_data['/network/end_time_gc'][...]  # needed to use bestNR
+
+    # Staistics value
+    filter_stats = ['coherent_snr', 'reweighted_snr', 'null_snr']
+    for param in filter_stats:
+        found_data[param] = inj_data['/network/%s' % param][...]
+    found_data['chisq'] = inj_data['/network/my_network_chisq'][...]
+    found_data['bank_chisq'] = numpy.zeros(len(found_data['eff_dist']))
+    found_data['cont_chisq'] = numpy.zeros(len(found_data['eff_dist']))
+    found_data['nifos'] = inj_data['/network/nifo'][...].astype(int)
+    # Calibration errors:
+    # Get the relative detector sensitivities averaged over the
+    # parameters. This is used to marginalize over calibration errors.
+    # also save the snr per ifo
+    snr_ifos = {'V1': 'v', 'H1': 'h1', 'L1': 'l'}
+    for ifo in ifos:
+        found_data['sigmasq_%s' % ifo] = inj_data['%s/sigmasq' % ifo][...]
+        found_data['snr_%s' % ifo] = inj_data[ifo + '/snr_' +
+                                              snr_ifos[ifo]][...]
+    # BestNRs
+    found_data['bestnr'] = \
+        get_bestnrs(found_data,
+                    ifos,
+                    snr_threshold=opts.snr_threshold,
+                    sngl_snr_threshold=opts.sngl_snr_threshold,
+                    new_snr_threshold=opts.newsnr_threshold)
+    if background_bestnrs is not None:
+        found_data['fap'] = numpy.array(
+                [sum(background_bestnrs > bestnr) for bestnr in
+                 found_data['bestnr']],
+                dtype=float) / len(background_bestnrs)
+    # Antenna responses
+    f_resp = {}
+    for ifo in ifos:
+        if sum(found_data['sigmasq_%s' % ifo] == 0):
+            logging.info("%s: sigmasq not set for at least one trigger.", ifo)
+        if sum(found_data['sigmasq_%s' % ifo] != 0) == 0:
+            logging.info("%s: sigmasq not set for any trigger.", ifo)
+            if len(ifos) == 1:
+                msg = "This is a single ifo analysis. "
+                msg += "Setting sigmasq to unity for all triggers."
+                logging.info(msg)
+                found_data['sigmasq_%s' % ifo][:] = 1.0
+        antenna = Detector(ifo)
+        f_resp[ifo] = get_antenna_responses(antenna, found_data['ra'],
+                                            found_data['dec'],
+                                            found_data['time'])
+
+    inj_sigma_mult = \
+        numpy.asarray([f_resp[ifo] *
+                       found_data['sigmasq_%s' % ifo] for ifo in ifos])
+    inj_sigma_tot = numpy.sum(inj_sigma_mult, axis=0)
+    for ifo in ifos:
+        found_data['inj_sigma_mean_%s' % ifo] = numpy.mean(
+            found_data['sigmasq_%s' % ifo] * f_resp[ifo] / inj_sigma_tot)
+    return found_data, missed_data
+
+
 def load_injections(inj_file, vetoes, sim_table=False, label=None):
```

Comment:
You can safely remove this function now and all imports that it required (if they do not break other things)